### PR TITLE
arastorage : add missing filepath and description to doxygen for arastorage.h

### PR DESCRIPTION
**Pull request by jeongarmy**
Add missing filepath and description in doxygen for arastorage.h to show this in File List of API page

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: framework/include/arastorage/arastorage.h [hunk 1]
```diff
@@ -53,11 +53,16 @@
 
 /**
  * @defgroup ARASTORAGE ARASTORAGE
- * @brief Provides APIs for Lightweight Database
+ * @brief Provides APIs for Arastorage Lightweight Database
```
Comment (sunghan-chang) [inline framework/include/arastorage/arastorage.h]: How about below?
@brief Provides APIs for Lightweight Database, Arastorage

File: framework/include/arastorage/arastorage.h [hunk 2]
```diff
@@ -53,11 +53,16 @@
...
  * @ingroup ARASTORAGE
  * @{
  */
 
+/**
+ * @file arastorage/arastorage.h
+ * @brief APIs for Arastorage Lightweight Database
```
Comment (sunghan-chang) [inline framework/include/arastorage/arastorage.h]: same as previous

File: framework/include/arastorage/arastorage.h (hunk 1 above)
Comment (jeongarmy) [reply framework/include/arastorage/arastorage.h]: It looks good, I will change

File: framework/include/arastorage/arastorage.h (hunk 2 above)
Comment (jeongarmy) [reply framework/include/arastorage/arastorage.h]: It looks good, I will change

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**